app/src/main/res/values/strings.xml

```diff
@@ -76,8 +76,9 @@
     <string name="importExport">Import/Export</string>
     <string name="importName">Import</string>
     <string name="exportName">Export</string>
-    <string name="importExportHelp">Data is imported to/exported from BudgetWatch.csv on external storage</string>
+    <string name="importExportHelp">Data is imported to/exported from BudgetWatch.<xliff:g id="extension">%s</xliff:g> on external storage</string>
     <string name="importedFrom">Imported from: %1$s</string>
+    <string name="fileFormat">File format</string>
```

Comment:
@paulle  This is the only string that needs to be translated, namely "fileFormat". If it is easy, you can simply respond in a comment with the translation and I'll handle putting it in the correct place.